lib/rubocop/cop/lint/literal_in_interpolation.rb

```diff
@@ -21,7 +21,14 @@ def on_dstr(node)
             next if special_keyword?(final_node)
             next unless LITERALS.include?(final_node.type)
 
-            add_offense(final_node, :expression)
+            add_offense(node, :expression)
```

Comment:
This change will change the message highlight. Guess this cop didn't have specs for the highlights.